ElectronClient/app/app.js

```diff
@@ -512,12 +512,20 @@ class Application extends BaseApplication {
 
 		function _showAbout() {
 			const p = packageInfo;
+			let gitInfo = '';
+			if (("git" in p) && ("tag" in p['git'] === false || !p.git.tag.includes(p.version))) {
+				gitInfo = "\n" + p.git.branch + ' (' + p.git.hash + ')';
+			}
 			let message = [
 				p.description,
 				'',
 				'Copyright © 2016-2019 Laurent Cozic',
 				_('%s %s (%s, %s)', p.name, p.version, Setting.value('env'), process.platform),
+				gitInfo
```

Comment:
Since gitInfo is not always set I would add it conditionally to the message array: `if (gitInfo) message.push(gitInfo)`. That way it won't leave a blank line when `message.join('\n')` is called below.

Reply:
When I tested it, it looked like the empty line was not added at the bottom, but maybe I didn't see it correctly.
I'll change it as requested.

Reply:
hmm, actually it is set, because it is initialized to an empty string. I will have to change the code a bit to use the `message.push`. Will do in a few minutes.

Reply:
`if (gitInfo)` will work if gitInfo is either "true" or an not an empty string. You can also use `!!gitInfo` to cast to a boolean.